pkg/services/searchV2/index.go

```diff
@@ -81,6 +85,17 @@ func (i *dashboardIndex) run(ctx context.Context) error {
 	}
 	i.logger.Info("Indexing for main org finished", "mainOrgIndexElapsed", time.Since(started), "numDashboards", len(dashboards))
 
+	// build bluge index
+	go func() {
+		orgID := int64(1)
```

Comment:
should we add a deferred func with a `recover()` while we are still testing out this approach?